build.sbt

```diff
@@ -37,6 +43,7 @@ lazy val parserCombinators = crossProject(JVMPlatform, JSPlatform, NativePlatfor
     libraryDependencies += "com.novocode" % "junit-interface" % "0.11" % Test
   )
   .jsSettings(
+    crossScalaVersions -= "0.22.0-RC1",
```

Comment:
To confirm: I assume this is not necessary for CI given the excludes in `.travis.yml`, but it's useful for local development?

Reply:
yes, looks like sbt-travisci doesn't understand the exclusion yet.